src/app/shared/form/builder/ds-dynamic-form-ui/models/scrollable-dropdown/dynamic-scrollable-dropdown.component.ts

```diff
@@ -79,6 +79,10 @@ export class DsDynamicScrollableDropdownComponent extends DsDynamicVocabularyCom
       tap(() => this.loading = false)
     ).subscribe((list: PaginatedList<VocabularyEntry>) => {
       this.optionsList = list.page;
+        if (this.model.value) {
+          this.setCurrentValue(this.model.value, true);
+        }
```

Comment:
This call is now inside the `loadOptions()` method which can be called multiple times. The original code only made this call during initialization, in the `ngOnInit()` method. I would add a change to make this call only during initialization, perhaps by adding a new boolean parameter to `loadOptions()` to indicate that it is the call made from `ngOnInit()`